### PR TITLE
fix: border and layout issues for status dialog

### PR DESCRIPTION
**Pull request by just-toby**
fixes some incorrect layout for the status dialog

### CODE REVIEW

File: src/components/Dialog.tsx [hunk 1]
```diff
@@ -66,8 +66,6 @@ export function Provider({ value, children, options }: ProviderProps) {
   // If a Dialog is active, mark the main content inert
   const ref = useRef<HTMLDivElement>(null)
   const [active, setActive] = useState(false)
-  // If pageCentered dialogs are enabled, we should mount them directly to the document body
-  // and adjust the styling in the renderer below so that they are centered and sized correctly.
```
Comment (JFrankfurt) [inline src/components/Dialog.tsx:70]: confirming this comment isn't relevant any longer

File: src/components/Dialog.tsx [hunk 2]
```diff
@@ -163,6 +161,7 @@ export const Modal = styled.div<{ color: Color; constrain?: boolean }>`
   height: ${({ constrain }) => (constrain ? 'fit-content' : '100%')};
   left: 0;
   outline: ${({ theme }) => `1px solid ${theme.outline}`};
+  outline: transparent;
```
Comment (JFrankfurt) [inline src/components/Dialog.tsx:164]: the second outline value here seems wrong

File: src/components/Dialog.tsx [hunk 3]
```diff
@@ -331,11 +335,8 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
       <ThemeProvider>
         <PopoverBoundaryProvider value={popoverRef.current} updateTrigger={updatePopover}>
           <div ref={popoverRef}>
-            <FullScreenWrapper enabled={pageCentered} onClick={onClose}>
-              <HiddenWrapper
-                constrain={pageCentered}
-                hidden={context.options?.animationType === DialogAnimationType.SLIDE}
-              >
+            <FullScreenWrapper enabled={pageCentered} onClick={pageCentered ? onClose : () => null}>
```
Comment (JFrankfurt) [inline src/components/Dialog.tsx:338]: I think this should continue to just be `onClose` and if you need a boolean behavior switch it should happen in that handler (or a wrapper, if that's a more applicable implementation)

File: src/components/Swap/Status/StatusDialog.tsx [hunk 4]
```diff
@@ -34,32 +34,22 @@ function TransactionStatus({ tx, onClose }: TransactionStatusProps) {
...
-    }
-    return null
-  }, [tx.receipt?.status])
-
   return (
-    <Column flex padded gap={0.75} align="stretch" style={{ height: '100%' }} data-testid="status-dialog">
+    <Column flex padded align="stretch" style={{ height: '100%', marginTop: '3em' }} data-testid="status-dialog">
       <StatusHeader icon={Icon} iconColor={tx.receipt?.status ? 'success' : undefined}>
-        <ThemedText.H4>{heading}</ThemedText.H4>
+        <ThemedText.H4 margin="3em 0 0">{heading}</ThemedText.H4>
```
Comment (JFrankfurt) [inline src/components/Swap/Status/StatusDialog.tsx:40]: nit: pref `styled(Component)` wrapping over inline style

File: src/components/Dialog.tsx [hunk 5]
```diff
@@ -320,7 +323,8 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
           return (mountPoint?.childElementCount ?? 0) > 1 ? SlideAnimationType.PAGING : SlideAnimationType.CLOSING
       }
     },
-    modal
+    modal,
+    context.options?.animationType === DialogAnimationType.NONE
```
Comment (JFrankfurt) [inline src/components/Dialog.tsx:327]: nit: would prefer not doing the boolean evaluation here and instead assign it to a variable and pass that it

**Review by JFrankfurt — CHANGES_REQUESTED** [review]

File: src/components/Dialog.tsx (hunk 1 above)
Comment (just-toby) [reply src/components/Dialog.tsx]: just wanted to move it, sorry for the confusion

File: src/components/Dialog.tsx (hunk 2 above)
Comment (just-toby) [reply src/components/Dialog.tsx]: thx

File: src/components/Dialog.tsx (hunk 5 above)
Comment (just-toby) [reply src/components/Dialog.tsx]: done

File: src/components/Swap/Status/StatusDialog.tsx (hunk 4 above)
Comment (just-toby) [reply src/components/Swap/Status/StatusDialog.tsx:40]: agree, but `ThemedText.H4` would overwrite it unless you use the prop - this is what the prop is intended for (similar to `Row`)

File: src/components/Dialog.tsx (hunk 3 above)
Comment (just-toby) [reply src/components/Dialog.tsx]: updated

**Review by JFrankfurt — APPROVED** [review]
LGTM

**Pull request merged**